packages/forma-36-react-components/src/components/Card/AssetCard/AssetCard.tsx

```diff
@@ -134,14 +126,7 @@ export class AssetCard extends Component<AssetCardProps> {
           <AssetCardSkeleton />
         ) : (
           <React.Fragment>
-            {withDragHandle && (
-              <CardDragHandle
-                isDragActive={isDragActive}
-                {...cardDragHandleProps}
-              >
-                Reorder entry
-              </CardDragHandle>
-            )}
+            {cardDragHandleComponent && cardDragHandleComponent}
```

Comment:
Just `{cardDragHandleComponent}` should be sufficient.

Reply:
@suevalov nice, updated